addon/-private/system/relationships/state/relationship.js

```diff
@@ -266,6 +269,33 @@ export default class Relationship {
     this.flushCanonicalLater();
   }
 
+  unloadDeletedInverseInternalModel(internalModel) {
```

Comment:
Per our discussion out of band, let's fix the inconsistency in naming here.  `unloadDeletedInverseInternalModel` and `unloadDeletedInverseInternalModelFromOwn` both mention an "inverse internal model" but are referring to different sides of the relationship.